frontend/src/components/header-views/header-content-course/HeaderContentCourse.js

```diff
@@ -39,6 +39,8 @@ class HeaderContentCourse extends Component {
 
   render() {
 
+    const displayCourseHeaderGraph = false;
```

Comment:
Yes please! The graph is broken in my opinion, thanks for removing it!

I should've reported the bug earlier, but for whatever reason I didn't.